### ISSUE TITLE
'show series' response breaking api change causing empty series list

### ISSUE THREAD

**Issue by rafaltrojniak** [vimeo/graphite-influxdb#52]
Hello,

Influxdb on 0.12 includes breaking api change in 'show swries' response.
https://docs.influxdata.com/influxdb/v0.11/concepts/010_vs_011
That causes problem  in the assure_series function.
I had temporary fixed the problem by modifying response parsing at the end of the function
```diff
-                 series = [key_name for (key_name,_) in ret.keys()]
+                 series = [key_name for [key_name] in  ret.raw['series'][0]['values']]
```

That should be probably configurable and/or detectable by the client (maybe client library?) and appropriate parsing function should be used.

**Comment by leoleovich** [comment]
@rafaltrojniak I sent a pull request with this change. Spent quite some time to write it and then seen your message.
https://github.com/vimeo/graphite-influxdb/pull/53/files

**Comment by gideon-maina** [comment]
I have been struggling with this error for a day and this did it for me. @leoleovich and @rafaltrojniak I have tested this fix with the latest version of influxdb (0.13) and it works as expected.

**Comment by Dieterbe** [comment]
merged the PR. thanks
